### ISSUE TITLE
Responsive Nav-Bar

### ISSUE THREAD

**Issue by Princerey** [Web3-Bharat-ITER/UI-Component-Library#25]
Hey @shashank-iter I want to add a new UI component for responsive Nav-bar. Could you please assign it to me .

**Comment by shashank-iter** [comment]
@Princerey Yeah sure, you can do that, kindly read the contribution guidelines and add a screenshot of your work while making a PR.

**Comment by Princerey** [comment]
Sure Thanks

**Comment by Princerey** [comment]
Please Check once I have created the PR.

**Comment by Aniketmishra0** [comment]
Hello Sir,
Can You assign this issue to me?

**Comment by SattyamSamania** [comment]
Hey @shashank-iter @Jyo561Python If the issue is still open can I work on this ?

**Comment by SattyamSamania** [comment]
Hey @shashank-iter I Have created a responsive navbar using Html and css. Should I make a PR of this. Have a look at it: 
![Navbar](https://user-images.githubusercontent.com/67833888/196974149-9c5b2b0e-ebb7-4f03-8268-16afb323e780.png)

**Comment by shashank-iter** [comment]
Looks good. Make a PR. @SattyamSamania

**Comment by SattyamSamania** [comment]
Hey @shashank-iter thanks for the suggestion. I have raised PR #126. Please check it.
